docs/getting_started.rst

```diff
@@ -405,3 +405,127 @@ the installer may use the ``installscript.qs`` script to make additional changes
+
+This time, ``aqt install-qt`` will only install one archive instead of the 27
+archives it installs by default.
+
+Let's say that ``qtbase`` is missing a few features that you need.
+To print a list of all the archives that are part of the base Qt installation,
+we can use the ``aqt list-qt --archives`` command:
+
+.. code-block:: console
+
+    $ aqt list-qt linux desktop --archives 5.15.2 gcc_64
+    icu qt3d qtbase qtconnectivity qtdeclarative qtgamepad qtgraphicaleffects qtimageformats
+    qtlocation qtmultimedia qtquickcontrols qtquickcontrols2 qtremoteobjects qtscxml
+    qtsensors qtserialbus qtserialport qtspeech qtsvg qttools qttranslations qtwayland
+    qtwebchannel qtwebsockets qtwebview qtx11extras qtxmlpatterns
+
+Notice what happened here: We used the ``--archives`` flag with two arguments:
+the version of Qt we are interested in, and the architecture we are using.
+As a result, the command printed a list of archives that are part of the base
+(non-minimal) Qt installation.
+
+Let's say we need to use ``qtmultimedia``, ``qtdeclarative``, ``qtsvg``, and
+nothing else. Remember that the ``qtbase`` archive is required for a minimal
+working Qt installation. We can install these archives using this command:
+
+.. code-block:: console
+
+    $ aqt install-qt linux desktop 5.15.2 --archives qtbase qtmultimedia qtdeclarative qtsvg
+
+Now let's say we need to install the ``qtcharts`` and ``qtlottie`` modules.
+Let's see what archives are part of these modules:
+
+.. code-block:: console
+
+    $ aqt list-qt linux desktop --archives 5.15.2 gcc_64 qtcharts qtlottie
+    qtcharts qtlottie
+
+This time, the command only printed two archives.
+The ``qtcharts`` and ``qtlottie`` modules are comprised of a single archive per module.
+Notice that the command printed all archives associated with the modules we provided it,
+and it did not print any of the archives associated with the base Qt installation.
+The ``--archives`` flag with a Qt version and architecture alone will print
+archives for the base Qt installation and none of the modules.
+When you add a list of modules to the command, it will print a list of all archives
+associated with those modules.
+
+Now let's say we need to install the ``debug_info`` module, which is particularly large.
+We do not want to install all of it, so we can use the ``--archives`` flag again
+to print which archives are part of the ``debug_info`` module:
+
+.. code-block:: console
+
+    $ aqt list-qt linux desktop --archives 5.15.2 gcc_64 debug_info
+    qt3d qtbase qtcharts qtconnectivity qtdatavis3d qtdeclarative qtgamepad qtgraphicaleffects
+    qtimageformats qtlocation qtlottie qtmultimedia qtnetworkauth qtpurchasing qtquick3d
+    qtquickcontrols qtquickcontrols2 qtquicktimeline qtremoteobjects qtscript qtscxml qtsensors
+    qtserialbus qtserialport qtspeech qtsvg qttools qtvirtualkeyboard qtwayland qtwebchannel
+    qtwebengine qtwebglplugin qtwebsockets qtwebview qtx11extras qtxmlpatterns
```

Comment:
There's a name collision between the `debug_info` archives and the archives in every other module/Qt base install. As I understand it, if you install `debug_info` and `qtcharts`, and specify `aqt install-qt --archives qtcharts`, you will install the `qtcharts` archives from both the `qtcharts` module and the `debug_info` module.

I think this name collision makes the feature harder to understand. I tried to explain it below, but I'm sure this is good enough.